app/services/cfe/service_set.rb

```diff
@@ -0,0 +1,52 @@
+module CFE
+  class ServiceSet
+    attr_reader :object
+
+    PASSPORTED_SERVICES = [
+      CreateAssessmentService,
+      CreateProceedingTypesService,
+      CreateApplicantService,
+      CreateCapitalsService,
+      CreateVehiclesService,
+      CreatePropertiesService,
+      CreateExplicitRemarksService,
+      ObtainAssessmentResultService,
+    ].freeze
+
+    NON_PASSPORTED_SERVICES = [
+      CreateAssessmentService,
+      CreateProceedingTypesService,
+      CreateApplicantService,
+      CreateCapitalsService,
+      CreateVehiclesService,
+      CreatePropertiesService,
+      CreateExplicitRemarksService,
+      CreateDependantsService,
+      CreateOutgoingsService,
+      CreateStateBenefitsService,
+      CreateOtherIncomeService,
+      CreateIrregularIncomesService,
+      CreateEmploymentsService,
+      CreateCashTransactionsService,
+      ObtainAssessmentResultService,
```

Comment:
Is the order these are called important?

I was wondering if it would be worthwhile defining a common set of services still, but I don't mind us being explicit either.

Reply:
after the third set goes in we could possibly refactor this again yes. There is also something in progress to perform call all but the `CreateAssessmentService` asynchronously to speed things up. The common set would be 

```
      CreateAssessmentService,
      CreateProceedingTypesService,
      CreateApplicantService,
      CreateCapitalsService,
      CreateVehiclesService,
      CreatePropertiesService,
      CreateExplicitRemarksService,
```

Reply:
the order is important, but only insofar as the `CreateAssessmentService` must go first i believe. Other may no different - @stephenrichards?

Reply:
👍 i will need to lock the test down as the test is not order sensitive but CFE is

Reply:
order sensitive now